tests/Doctrine/MongoDB/Tests/CursorTest.php

```diff
@@ -381,6 +381,44 @@ public function testRecreate()
         $cursor->recreate();
     }
 
+    public function testSetMaxTimeMSWhenRecreateCursor()
+    {
+        if (version_compare(phpversion('mongo'), '1.5.0', '<')) {
+            $this->markTestSkipped('This test is not applicable to driver versions < 1.5.0');
```

Comment:
If we use `addOption()`, I think we can run this test on any driver version with appropriate mock changes.